src/views/EventPage.vue

```diff
@@ -0,0 +1,266 @@
+<template>
+  <div class="page-container">
+    <!-- background img -->
+    <div class="background-img">
+      <div class="gradient-overlay" />
+      <img :src="event.mediaLink.cover" alt class="background-img" />
```

Comment:
There needs to be a default image (maybe start an issue, or just solve here), no image results in white background, which makes the top buttons not visible
<img width="398" alt="Screen Shot 2020-03-10 at 12 12 05 AM" src="https://user-images.githubusercontent.com/44561376/76362857-12bfa500-62df-11ea-9a61-21c6119082c0.png">

Reply:
ok so i made the nav buttons turn dark when there is no image 👌👌 in the future tho we might want to set a boolean for light/dark mode maybe in the event object so the nav buttons can adapt, although i'm still not sure how to change the css accordingly

Reply:
for the text i guess we can just use a conditional as an inline style

Reply:
OR we can have two of the elements for each thing, and v if depending on the value of the boolean

Reply:
This was the difficult and complicated method I wanted to implement LOL. If we plan to do this, then probably refrain from doing any temporary 'fixes' (or make sure to include a `// TODO` to remind later.

For my own sanity I made a 'TODO' issue that consists of some of the issues I know I will forget to do. Look it up [here](https://github.com/onlinkers/event-discovery-api/issues/11)

Reply:
yeah in the long run i think we should go with no.2 i suggested in the group -- having one page for event and ideas (referring to the event ideation page design on figma)